### DISCUSSION TITLE
HOT TV [NSFW]

### DISCUSSION THREAD

**Discussion by freearhey** [iptv-org/community#528] [Q&A]
Couldn't find any information about these channels.

HOT 1: http://85.11.144.9:4222/SuperOne?token=test
HOT 2: http://85.11.144.9:4222/ExoticaPlus?token=test

On-screen bugs:
<img width="54" alt="image" src="https://github.com/iptv-org/iptv-org.github.io/assets/7253922/f3e03098-73b6-472c-b32e-301c6b1c6a3b">
<img width="50" alt="image" src="https://github.com/iptv-org/iptv-org.github.io/assets/7253922/66e14822-b0d9-4db8-b937-f55e86eada1a">

If you know what country these channels are broadcast from or where to find the logo, let me know so I can add them to the database and playlist.

**Comment by BellezaEmporium** [comment]
Sounds like bootleg Russian channels, weirdly enough.

The content being aired comes from different companies. Not your "classic NSFW channel" per-se.
